models/movies.js

```diff
@@ -0,0 +1,42 @@
+const { Sequelize, Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Movies extends Model {}
+
+Movies.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    movie_name: {
+      type: DataTypes.STRING,
+    }
+    movie_description: {
+      type: DataTypes.STRING,
+    },
+    {
+      user_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
```

Comment:
a table can have only 1 primary key and we don't want it to auto increment since we're just pulling the existing user_id, not making a new one

Reply:
Got it! Thanks!